### PR TITLE
docs: configuring access to Kubernetes and OpenShift clusters

### PR DESCRIPTION
**Pull request by themr0c**
Added procedures describing how to configure access to Kubernetes and OpenShift clusters:

* Configuring access to a Kubernetes cluster.
* Creating an OpenShift Local cluster
* Configuring access to a Developer Sandbox
* Configuring access to an OpenShift cluster

References: https://github.com/containers/podman-desktop/issues/1449
Followup for: https://github.com/containers/podman-desktop/pull/2303

### CODE REVIEW

File: website/docs/kubernetes/openshift/configuring-access-to-a-developer-sandbox.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+#### Procedure
+
+1. Install the _Developer Sandbox_ extension: go to **Dashboard**, and click **Developer Sandbox <icon icon="fa-solid fa-download" size="lg" />**.
+1. Go to **<icon icon="fa-solid fa-cog" size="lg" /> Settings > Resources**.
+1. In the **Developer Sandbox** tile, click **Create new**.
+1. In the **Create a Developer Sandbox** screen, click **Log into Developer Sandbox**.
+1. In the **Open external website** dialog, click **Yes**.
+1. In the [Developer Sandbox website](https://developers.redhat.com/developer-sandbox):
+   1. Click **Start your sandbox for free**.
```
Comment (jeffmaury) [inline website/docs/kubernetes/openshift/configuring-access-to-a-developer-sandbox.md]: If it's the first time you may end up with a verification workflow. The whole workflow does not need to be specified but at least it should be mentioned

File: website/docs/kubernetes/openshift/configuring-access-to-an-openshift-cluster.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+With Podman Desktop, you can configure access to your OpenShift cluster.
+
+#### Prerequisites
+
+- You have credentials for your OpenShift cluster.
+
+#### Procedure
+
+1. Go to **OpenShift**.
```
Comment (jeffmaury) [inline website/docs/kubernetes/openshift/configuring-access-to-an-openshift-cluster.md]: I don't see where it is ?

File: website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+   <TabItem value="mac" label="macOS">
+
+   1. In the **OpenShift Local** tile, click **<icon icon="fa-solid fa-rocket" size="lg" /> Install**.
+   1. When prerequisites are missing, follow the instructions.
+   1. In the **Red Hat OpenShift Local** screen, click **Yes** to start the installation.
+   1. Follow the installation program instructions.
+   1. Reboot to finalize system changes.
+
+   </TabItem>
+   <TabItem value="linux" label="Linux">
```
Comment (jeffmaury) [inline website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md:46]: Don't we support installation on Linux through the extension ?

File: website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+      $ crc setup
+      ```
+
+   1. Exit and restart Podman Desktop.
+
+   </TabItem>
+   </Tabs>
+
+1. (Optionally) Review the extension settings in **<icon icon="fa-solid fa-cog" size="lg" /> Settings > Preferences > Extension: Red Hat OpenShift Local**.
+1. On the **Dashboard**, click **Initialize and start**.
```
Comment (jeffmaury) [inline website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md:65]: Initialize and start never asked me (Windows) to select a preset ? Is this new ?

File: website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+   1. Select your OpenShift Local Virtual machine preset, if not set in **<icon icon="fa-solid fa-cog" size="lg" /> Settings > Preferences > Extension: Red Hat OpenShift Local > Preset**.
+      - _Microshift_ (experimental): provides a lightweight and optimized environment with a limited set of services.
```
Comment (jeffmaury) [inline website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md]: I think it's MicroShift

File: website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md (hunk 3 above)
Comment (themr0c) [reply website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md:46]: No (confirmed in chat by @gbraad)

File: website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md (hunk 4 above)
Comment (themr0c) [reply website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md:65]: It does only ask if it is not already set in your settings. It asked to me on Windows, once :).

File: website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md (hunk 5 above)
Comment (themr0c) [inline website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md]:
```suggestion
      - _MicroShift_ (experimental): provides a lightweight and optimized environment with a limited set of services.
```

File: website/docs/kubernetes/openshift/configuring-access-to-an-openshift-cluster.md (hunk 2 above)
Comment (themr0c) [reply website/docs/kubernetes/openshift/configuring-access-to-an-openshift-cluster.md]: Ah! Good point. Maybe it's something specific with my installation. I will check.

File: website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md (hunk 3 above)
Comment (praveenkumar) [reply website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md:46]: We don't have any installer for Linux.

File: website/docs/kubernetes/openshift/configuring-access-to-an-openshift-cluster.md (hunk 2 above)
Comment (themr0c) [reply website/docs/kubernetes/openshift/configuring-access-to-an-openshift-cluster.md]: I have this on my Linux host:

Screenshot from 2023-05-17 10:43:53 Selection 1829x523 01

Comment (themr0c) [reply website/docs/kubernetes/openshift/configuring-access-to-an-openshift-cluster.md]: Oh, it's the Docker extension!

Screenshot from 2023-05-17 10:45:20 Selection 736x688 01

Should I add it to the docs, or should we just leave it out?

Comment (jeffmaury) [reply website/docs/kubernetes/openshift/configuring-access-to-an-openshift-cluster.md]: I don't think so.

File: website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md (hunk 4 above)
Comment (themr0c) [reply website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md:65]: It only happens when you do a new install (uninstall and try again).

Comment (gbraad) [reply website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md:65]: Right, this is an onboarding step. Not a recurring event.

File: website/docs/kubernetes/openshift/configuring-access-to-an-openshift-cluster.md (hunk 2 above)
Comment (themr0c) [reply website/docs/kubernetes/openshift/configuring-access-to-an-openshift-cluster.md]: That was the OpenShift extension for Docker. I removed it from my install and from this pull request. We don't need to document it.

File: website/docs/kubernetes/openshift/configuring-access-to-a-developer-sandbox.md (hunk 1 above)
Comment (themr0c) [reply website/docs/kubernetes/openshift/configuring-access-to-a-developer-sandbox.md]: Added.

**Comment by themr0c** [comment]
@gbraad @praveenkumar @jeffmaury I believe I addressed all comments. Can you review again?

File: website/docs/kubernetes/openshift/configuring-access-to-a-developer-sandbox.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+sidebar_position: 1
+title: Developer Sandbox
+description: Configuring access to a Developer Sandbox
+keywords: [podman desktop, podman, containers, pods, migrating, kubernetes, openshift]
+tags: [migrating-to-kubernetes, openshift]
+---
+
+# Configuring access to a Developer Sandbox
+
+The [Developer Sandbox](https://developers.redhat.com/developer-sandbox) is a free, private OpenShift environment including one project and a resource quota of 14 GB RAM, and 40 GB storage.
```
Comment (jeffmaury) [inline website/docs/kubernetes/openshift/configuring-access-to-a-developer-sandbox.md:11]: The FAQ (https://developers.redhat.com/developer-sandbox/FAQ) has different values. Where did you get them from ?

File: website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+   1. Exit and restart Podman Desktop.
+
+   </TabItem>
+   </Tabs>
+
+1. (Optionally) Review the extension settings in **<icon icon="fa-solid fa-cog" size="lg" /> Settings > Preferences > Extension: Red Hat OpenShift Local**.
+1. On the **Dashboard**, click **Initialize and start**.
+
+   1. Select your OpenShift Local Virtual machine preset, if not set in **<icon icon="fa-solid fa-cog" size="lg" /> Settings > Preferences > Extension: Red Hat OpenShift Local > Preset**.
```
Comment (jeffmaury) [inline website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md:67]: I'm not fully sure I was ever asked for select the preset on Windows

File: website/docs/kubernetes/openshift/configuring-access-to-a-developer-sandbox.md (hunk 6 above)
Comment (themr0c) [reply website/docs/kubernetes/openshift/configuring-access-to-a-developer-sandbox.md:11]: I copied this from somewhere. If not this page, something displayed by the extension.

File: website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md (hunk 7 above)
Comment (praveenkumar) [reply website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md:67]: To me on the mac from a clean install it did ask me to select the preset.

Comment (gbraad) [reply website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md:67]: You are only asked this on a clean install... it is part of 'onboarding'.

File: website/docs/kubernetes/openshift/configuring-access-to-a-developer-sandbox.md (hunk 6 above)
Comment (gbraad) [reply website/docs/kubernetes/openshift/configuring-access-to-a-developer-sandbox.md:11]: Perhaps not specifying them, but referring to an online resource with up-to-date information?

Comment (jeffmaury) [reply website/docs/kubernetes/openshift/configuring-access-to-a-developer-sandbox.md:11]: Sandbox team confirmed their FAQ is not up to date and resources are the one listed here

File: website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md (hunk 7 above)
Comment (jeffmaury) [reply website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md:67]: Yes but as you have to reboot....

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**